### ISSUE TITLE
A11y(Angle Slider): ARIA input fields must have an accessible name

### ISSUE THREAD

**Issue by karim-semmoud** [chakra-ui/zag#2781]
# 🐛 Bug report

When testing A11y on Angle Slider, we received the following error "ARIA input fields must have an accessible name" on thumb part
```
 ● AngleSlider has no accessibility violations

    Expected value   undefined
    Received:
      undefined
    
    Message:
      expect(received).toHaveNoViolations(expected)
    
    Expected the HTML found at $('#angle-slider\:\:r0\:\:thumb') to have no violations:
    
    <div data-scope="angle-slider" data-part="thumb" id="angle-slider::r0::thumb" role="slider" aria-valuemax="360" aria-valuemin="0" aria-valuenow="0" tabindex="0"></div>
    
    Received:
    
    "ARIA input fields must have an accessible name (aria-input-field-name)"
    
    Fix any of the following:
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
    
```
## 💥 Steps to reproduce

Load the default react angle slider and run a11y test

## 💻 Link to reproduction

CodeSandbox reproduction: [https://stackblitz.com/edit/xwcvag2g?file=src%2FAngleSlider.a11y.test.tsx](https://stackblitz.com/edit/xwcvag2g?file=src%2FAngleSlider.a11y.test.tsx)

`npm test` to start the A11y test

## 🧐 Expected behavior

A11y should pass without violations

## 🧭 Possible Solution

Update Angle Slider Connect for Thumb part

## 🌍 System information

| Software         | Version(s) |
| ---------------- | ---------- |
| Zag Version      |       latest     |
| Browser          |     Chrome       |
| Operating System |     Linux       |

## 📝 Additional information

I noticed that e2e tests for angle slider are missing, let me know if you would like me to create some basic one based of the other components tests

**Comment by segunadebayo** [comment]
I just pushed a fix for this. We'll release an update shortly.

If the issue persists after upgrading, I'll re-open it.
